### ISSUE TITLE
python 3.5 f-string

### ISSUE THREAD

**Issue by sspitz3** [TDAmeritrade/stumpy#48]
Python 3.5 does not have f strings:

https://github.com/TDAmeritrade/stumpy/blob/master/stumpy/stomp.py#L131

Should stumpy be compatible with 3.5?

**Comment by seanlaw** [comment]
@sspitz3 Good catch! This is a conundrum. Technically, if the code runs without errors then it is compatible. However, as you pointed out, if it needs to print a logging error then it would fail. This is probably why all of the unit tests pass.

The simple solution would be to bump the minimum version up to Python 3.6+. Do you have any thoughts or suggestions?

**Comment by sspitz3** [comment]
For me, it is not compatible as I discovered the problem by trying to set up a Python 3.5 conda environment:

```
conda create --name stumpy-test python=3.5
conda activate stumpy-test
pip install stumpy
```
A python shell in this environment then fails on `import stumpy` due to the f-strings.

The environment for my use case is restricted to Python 3.5, so I am biased to changing the f-strings to use `format`.

**Comment by seanlaw** [comment]
Okay, let me take a look at the options.

For now, you can probably just clone the repo, change the text, and `pip install .` 

I'll try to fix this for the next release.

**Comment by seanlaw** [comment]
This probably isn't the response that you are looking for but this one will have to be summed up as a typo and we will only be able to support Python 3.6+. This is due to the fact that we are using black for code formatting and black is only available for Python 3.6+. Sorry for the inconvenience.
